README.md

```diff
@@ -1,291 +1,152 @@
 
-<!-- GETTING STARTED -->
-
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
-To get a local copy up and running, follow these steps.
+To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
 In order to run this project you need:
 
-<!--
-Example command:
+- Code Editor (Vs Code)
 
-```sh
- gem install rails
-```
- -->
+- Terminal
 
-### Setup
+- Install ruby on your computer
+
+- Install rails on your computer
 
-Clone this repository to your desired folder:
+### Setup
 
-<!--
-Example commands:
+Clone this repository by using the command line to your desired folder:  
 
 ```sh
-  cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+git clone https://github.com/Marcraphael12/rails_blog_app.git
+
+cd rails_blog_app
 ```
---->
 
 ### Install
 
 Install this project with:
 
-<!--
-Example command:
-
 ```sh
-  cd my-project
-  gem install
+cd rails_blog_app
+ bundle install (install all dependencies)
 ```
---->
 
 ### Usage
 
 To run the project, execute the following command:
 
-<!--
-Example command:
-
-```sh
-  rails server
-```
---->
-
-### Run tests
-
-To run tests, run the following command:
-
-<!--
-Example command:
-
 ```sh
-  bin/rails test test/models/article_test.rb
+rails server
 ```
---->
-
-### Deployment
-
-You can deploy this project using:
 
-<!--
-Example:
+### Test
 
 ```sh
-
+  rspec
 ```
- -->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- AUTHORS -->
-
-## 👥 Authors <a name="authors"></a>
-
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
+## Authors <a name="authors"></a>
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+👤 **Marc Raphael**
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FUTURE FEATURES -->
+- GitHub: [@Marcraphael12](https://github.com/Marcraphael12)
+- Twitter: [@Marcraphael20](https://twitter.com/MarcRaphael20)
+- LinkedIn: [Marc raphael](http://www.linkedin.com/in/marc-raphael-326039204)
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-> Describe 1 - 3 features you will add to the project.
-
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+- **Improve styles of the project**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- CONTRIBUTING -->
 
 ## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the [issues page](https://github.com/Marcraphael12/rails_blog_app/issues)
```

Comment:
- [ ] [Optional] The issues page link is broken in the README file. I would suggest to fix this. It will make the README file more professional.

Reply:
all good